### ISSUE TITLE
The app doesn't start 

### ISSUE THREAD

**Issue by mancubus77** [andreas-marschke/mojo-rickshaw#1]
I assume that the code is qute old, but nevertheless:
[Fri Apr 28 10:11:14 2017] [debug] Template "exception.development.html.ep" not found
[Fri Apr 28 10:11:14 2017] [debug] Template "exception.html.ep" not found
[Fri Apr 28 10:11:14 2017] [debug] Rendering cached template "mojo/debug.html.ep"
[Fri Apr 28 10:11:14 2017] [debug] Rendering cached template "mojo/menubar.html.ep"
[Fri Apr 28 10:11:14 2017] [debug] 500 Internal Server Error (0.018759s, 53.308/s)
[Fri Apr 28 10:11:15 2017] [debug] GET "/json"
[Fri Apr 28 10:11:15 2017] [debug] Routing to a callback
[Fri Apr 28 10:11:15 2017] [error] Can't locate object method "render_json" via package "Mojolicious::Controller" at ./mojo-h5bp line 17.

**Comment by mancubus77** [comment]
--- mojo-h5bp	(revision 5bbb7649e94a6f20aaffb6551d230536adecb58c)
+++ mojo-h5bp	(revision 90f7c429e83a702d04eac909f2986fdd60659a51)
@@ -14,7 +14,7 @@
 get '/json' => sub {
   my $self = shift;
   
-  $self->render_json({ "New York" => rand() * 100 ,
+  $self->render(json=>{ "New York" => rand() * 100 ,
 		       "Washington" =>  rand() * 100 ,
 		       "Miami" => rand() * 100
 		     });

**Comment by andreas-marschke** [comment]
Yup. It was just an experiment at the time. I don't personally work with mojolicious or rickshaw anymore. If you wish to pickup the repository and work on it yourself I can transfer it.
